gr-iio/CMakeLists.txt

```diff
@@ -26,7 +26,23 @@ include(GrComponent)
 GR_REGISTER_COMPONENT("gr-iio" ENABLE_GR_IIO ${GR_IIO_DEPENDENCIES})
 
 if(libad9361_FOUND)
-  GR_APPEND_SUBCOMPONENT("libad9361")
+  # Sadly, libad9361 is not reliably versioned (yet?).
+  # Especially Fedora ships an incompatible early version of the ad9361.h header
+  # Let's check whether things compile.
+  set(OLD_CMAKE_REQUIRED_INCLUDES ${CMAKE_REQUIRED_INCLUDES})
+  set(CMAKE_REQUIRED_INCLUDES ${libad9361_INCLUDE_DIRS})
+  CHECK_CXX_SOURCE_COMPILES("
+    #include <ad9361.h>
+    namespace testing
+    {
+      auto test_ad9361_multichip_sync = &::ad9361_multichip_sync;
+      // auto test_ad9361_set_bb_rate_custom_filter_manual = &::ad9361_set_bb_rate_custom_filter_manual;
```

Comment:
Should this comment be removed?

**Adam Horden**

Reply:
yes it should. But it wasn't. But it still didn't compile on my machine. Which means something is wrong :(